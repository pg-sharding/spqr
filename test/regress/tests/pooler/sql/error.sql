CREATE TABLE tt(i INT);

BEGIN;

INSERT INTO tt VALUES(1);

DROP TABLE tt;

INSERT INTO tt VALUES(1);
INSERT INTO tt VALUES(1);
INSERT INTO tt VALUES(1);
INSERT INTO tt VALUES(1);

ROLLBACK;
/* should exists */
TABLE tt;
