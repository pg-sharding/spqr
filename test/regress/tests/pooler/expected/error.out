CREATE TABLE tt(i INT);
BEGIN;
INSERT INTO tt VALUES(1);
DROP TABLE tt;
INSERT INTO tt VALUES(1);
ERROR:  relation "tt" does not exist
LINE 1: INSERT INTO tt VALUES(1);
                    ^
INSERT INTO tt VALUES(1);
ERROR:  current transaction is aborted, commands ignored until end of transaction block
INSERT INTO tt VALUES(1);
ERROR:  current transaction is aborted, commands ignored until end of transaction block
INSERT INTO tt VALUES(1);
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ROLLBACK;
/* should exists */
TABLE tt;
 i 
---
(0 rows)

